clientlib/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -460,10 +467,11 @@ message StatsResponse {
      * counted you should call {@link IndexWriter#commit()} first.
      */
     int32 numDocs = 3;
-    string state = 4; //state of the index
-    Taxonomy taxonomy = 5; //Taxonomy(facets) stats
-    repeated Searcher searchers = 6; //Searcher stats
-    Searcher currentSearcher = 7; //Current Searcher stats
+    int64 dirSize = 4; //size of the this indexDir
```

Comment:
we can add this new field to the stats test too

Reply:
fixed